<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDialog</class>
 <widget class="QDialog" name="FileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dostępne labirynty:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="tw_filesList">
     <column>
      <property name="text">
       <string>Nazwa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Wymiary</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>St. trudności</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Aktualny czas</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_fileName">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Wpisz nazwę labiryntu lub wybierz z listy</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Accept">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Delete">
       <property name="text">
        <string>Usuń</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Close">
       <property name="text">
        <string>Zamknij</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
